libpod/runtime_ctr.go

```diff
@@ -124,6 +124,7 @@ func (r *Runtime) newContainer(ctx context.Context, rSpec *spec.Spec, options ..
 	if err != nil {
 		return nil, errors.Wrapf(err, "error initializing container variables")
 	}
+	ctr.config.OSArguments = os.Args
```

Comment:
Can we pass this in with a With... function during container creation? I don't know if we want this for e.g. CRI-O containers.